### PR TITLE
BREAKING: update documentation by setting default video: false for v13.0.0

### PR DESCRIPTION
**Pull request by AtofStryker**
updates the video `true` configuration to `false` and other relevant documentation to the change. see https://github.com/cypress-io/cypress/pull/27008

### CODE REVIEW

**Review by chrisbreiding — APPROVED** [review]

File: docs/guides/end-to-end-testing/migration/protractor-to-cypress.mdx [hunk 1]
```diff
@@ -156,9 +156,9 @@ the state of the app UI at the point of failure or see the events that led up to
 the failure.
 
 Cypress assists with debugging in headless mode, by automatically taking a
-screenshot of the app UI and command log at the exact point of test failure. To
-help see everything that happened prior to test failure, Cypress provides a
-video recording (as an MP4 file) of a full test spec run by default.
+screenshot of the app UI and command log at the exact point of test failure. If
+enabled, to help see everything that happened prior to test failure, Cypress
+provides a video recording (as an MP4 file) of a full test spec run by default.
```
Comment (emilyrohrbough) [inline docs/guides/end-to-end-testing/migration/protractor-to-cypress.mdx:161]:
```suggestion
screenshot of the app UI and command log at the exact point of test failure. To see everything that happened prior to test failure, enabled the `video` configuration option to record (as an MP4 file) the full test spec run.
```
this isn't by default anymore

File: docs/guides/guides/screenshots-and-videos.mdx [hunk 2]
```diff
@@ -43,21 +43,28 @@ to clear your screenshots folder before a run, you can set
...
 ## Videos
 
-Cypress records a video for each spec file when running tests during
-`cypress run`. Videos are _not_ automatically recorded during `cypress open`.
+If enabled, Cypress records a video for each spec file when running tests during
+`cypress run`. Videos are _not_ recorded during `cypress open`.
 
-Video recording can be turned off entirely by setting
-[`video`](/guides/references/configuration#Videos) to `false` from within your
+Video recording can be turned on by setting
```
Comment (emilyrohrbough) [inline docs/guides/guides/screenshots-and-videos.mdx:49]:
```suggestion
Video recording is disabled by default, but can be turned on by setting
```

Comment (emilyrohrbough) [reply docs/guides/guides/screenshots-and-videos.mdx:49]: I think this should be the first sentence to flow these a bit better. its disabled > heres how you turn it on > when it's on, here is the behavior.

File: docs/guides/overview/why-cypress.mdx [hunk 3]
```diff
@@ -117,8 +117,8 @@ do that no other testing framework can:
 - **Consistent Results:** Our architecture doesn't use Selenium or WebDriver.
   Say hello to fast, consistent and reliable tests that are flake-free.
 - **Screenshots and Videos:** View screenshots taken automatically on failure,
-  or videos of your entire test suite when run from the CLI. Record to
-  [Cypress Cloud](/guides/cloud/introduction) to store them with your test
+  or videos, if enabled, of your entire test suite when run from the CLI. Record
+  to [Cypress Cloud](/guides/cloud/introduction) to store them with your test
   results for zero-configuration debugging.
```
Comment (emilyrohrbough) [inline docs/guides/overview/why-cypress.mdx:122]: This statement is no longer true. 
```suggestion
  results.
```

**Review by emilyrohrbough — CHANGES_REQUESTED** [review]

File: docs/guides/guides/screenshots-and-videos.mdx (hunk 2 above)
Comment (AtofStryker) [reply docs/guides/guides/screenshots-and-videos.mdx]: @emilyrohrbough how is 1b5358019?

**Review by emilyrohrbough — APPROVED** [review]

File: docs/guides/end-to-end-testing/migration/protractor-to-cypress.mdx [hunk 4]
```diff
@@ -156,9 +156,7 @@ the state of the app UI at the point of failure or see the events that led up to
 the failure.
 
 Cypress assists with debugging in headless mode, by automatically taking a
-screenshot of the app UI and command log at the exact point of test failure. To
-help see everything that happened prior to test failure, Cypress provides a
-video recording (as an MP4 file) of a full test spec run by default.
+screenshot of the app UI and command log at the exact point of test failure. To see everything that happened prior to test failure, enabled the `video` configuration option to record (as an MP4 file) the full test spec run.
```
Comment (AtofStryker) [inline docs/guides/end-to-end-testing/migration/protractor-to-cypress.mdx:159]:
```suggestion
screenshot of the app UI and command log at the exact point of test failure. To see everything that happened prior to test failure, enable the `video` configuration option to record (as an MP4 file) the full test spec run.
```

**Pull request merged**